app/konnect-platform/kong-identity.md

```diff
@@ -0,0 +1,154 @@
+* [OpenID Connect plugin](/plugins/openid-connect/)
+* [OAuth2.0 Introspection plugin](/plugins/oauth2-introspection/)
+* [Upstream OAuth plugin](/plugins/upstream-oauth/)
+
+## How Kong Identity works
+
+Kong Identity allows you to create auth servers, claims, scopes, and clients in {{site.konnect_short_name}} using the [{{site.konnect_short_name}} API](/api/konnect/kong-identity/v1/#/). Each of these components plays a specific role in how access is managed:
+* **Auth server:** Issue OAuth 2.0 and OpenID Connect tokens that you can use to authenticate a client (machine) with your Gateway Services. Each auth server is unique to your organization and [{{site.konnect_short_name}} region](/konnect-platform/geos/). We recommend creating different auth servers for different environments or subsidiaries.
+* **Clients:** Represent machines that request tokens, such as microservices, mobile apps, or automation scripts.
+* **Scopes:** Define what those clients are allowed to access. 
+* **Claims:** Optional pieces of metadata, like user roles or environment tags, that can be included in tokens and forwarded to upstream services.
+
+To use Kong Identity for authentication, you must configure one of the supported plugins (OpenID Connect, OAuth2.0 Introspection, or Upstream OAuth). These plugins determine how tokens are validated, introspected, or passed along to upstream services.
+
+## Kong Identity client credential authentication flow
+
+The following diagram shows how authentication works with Kong Identity:
+
+<!--vale off-->
+{% mermaid %}
+sequenceDiagram
+    autonumber
+    participant client as Client Application
+    participant idsvc as Kong Identity Service
+    participant gateway as Kong Gateway
+    participant api as Customer API
+    client->>idsvc: request access token<br>(Client ID, Client Secret, requested scope)
+    idsvc->>idsvc: validate credentials
+    idsvc-->>client: access token, granted scope, expiry time
+    client->>gateway: /get_protected_resource<br>(access token, granted scope)
+    gateway->>idsvc: validate access token
+    idsvc-->>gateway: validate access token
+    gateway->>api: get_resource
+    api-->>gateway: resource
+    gateway-->>client: resource
+{% endmermaid %}
+<!--vale on-->
+
+## Kong Identity authorization code flow
+
+In the authorization code flow:
+1. Optional: The client application displays the user consent page and authenticates the user (this part is handled outside {{site.base_gateway}}). When the user clicks **Authorize**, the client app calls the `/authorize` endpoint created by attaching the OAuth2 plugin to a service.
+   {:.info}
+   > If an app requires user authentication, the authorization step must happen outside of {{site.konnect_short_name}}.
+3. The request includes the client ID, secret, and scopes the user consented to.
```

Comment:
```suggestion
3. The client makes a request that includes the client ID, secret, and scopes the user consented to.
```
Making this an action rather than just info.